kube-app-testing.sh

```diff
@@ -859,12 +860,18 @@ run_pytest () {
   done
 
   info "Starting tests with pipenv+pytest, saving results to \"${test_res_file}\""
+  # if the tests are running against a KinD cluster then we want to use the internal
+  # config we generated earlier. if this isn't a KinD cluster then we just skip past
+  # and use the kubeconfig generated for external access
+  if [[ "${CLUSTER_TYPE}" ==  "kind" ]]
+    KUBECONFIG=${KUBECONFIG_I}
+  fi
   pipenv_cmd='PATH=$HOME/.local/bin:$PATH pipenv sync && PATH=$HOME/.local/bin:$PATH pipenv run pytest --full-trace --verbosity=8 .'
   docker run -it --rm \
     -v ${TMP_DIR}/.local:/root/.local \
     -v ${TMP_DIR}/.cache:/root/.cache \
     -v `pwd`:/chart -w /chart \
-    -v ${KUBECONFIG_I}:/kube.config:ro \
+    -v ${KUBECONFIG}:/kube.config:ro \
```

Comment:
Wait, have you tested this with kind? I think it won't work without `KUBECONFIG_I` passed here

Reply:
I haven't, no, but if you check out this line you can see that I substitute the internal kubeconfig in if we're running in kind https://github.com/giantswarm/kube-app-testing/pull/5/files#diff-ae1823197b8c0bbfdac99da865a7213eR866-R868